README.md

```diff
@@ -1 +1,51 @@
-# build-and-push-action
\ No newline at end of file
+# The Build and Push GitHub Action
+
+## Action Inputs
+
+### Registry information
+
+| Input Name           | Description                                                 | Default value |
+|----------------------|-------------------------------------------------------------|---------------|
+| `registry`           | Registry to push container image to.                        | **required**  |
+| `registry_namespace` | Namespace of the registry, where the image would be pushed. | **required**  |
+| `registry_username`  | Login to specified registry.                                | **required**  |
+| `registry_token`     | Access token to specified registry.                         | **required**  |
+
+### Build information
+
+| Input Name       | Description                                                     | Default value |
+|------------------|-----------------------------------------------------------------|---------------|
+| `version `       | Version of the built image.                                     | **required**  |
+| `tag`            | Tag of the built image.                                         | **required**  |
```

Comment:
Can this be field? like: ['latest', 'c9s', <datetime> ]

Reply:
I did not work with this concept, so IMO not. I will rework it.

Reply:
I double-checked the `tag` problem.

What is please the use-case of inserting multiple tags?

When building this action I used structure from build-and-push.yml from all sclorg containers, and in those is always stated only one tag as an input. Example of nodejs container:
```
 14           - version: "6"
 15             dockerfile: "Dockerfile"
 16             registry_namespace: "centos7"
 17             tag: "centos7"
 18           - version: "6"
 19             dockerfile: "Dockerfile.c9s"
 20             registry_namespace: "sclorg"
 21             tag: "c9s"
```